docs/site/Routes.md

```diff
@@ -10,83 +10,120 @@ summary:
+A `Route` is the mapping between your API specification and an Operation. It
+tells LoopBack which Operation to `invoke()` given an HTTP request.
 
 The `Route` object and its associated types are provided as a part of the
-[(`@loopback/rest`)](https://github.com/strongloop/loopback-next/blob/master/packages/rest) package.
+[`@loopback/rest`](https://github.com/strongloop/loopback-next/blob/master/packages/rest)
+package.
 
 ## Operations
 
-Operations are JavaScript functions that accept Parameters. They can be implemented as plain JavaScript functions or as methods in [Controllers](Controllers.md).
+Operations are functions that accept Parameters. They can be implemented as
+plain JavaScript/TypeScript functions (like http handler functions) or as
+methods in [Controllers](Controllers.md).
 
-```js
+```ts
 // greet is a basic operation
-function greet(name) {
-  return `hello ${name}`;
+function greet(name: string) {
+  return `hello ${name || user}`;
 }
 ```
 
 ## Parameters
 
-In the example above, `name` is a Parameter. Parameters are values, usually parsed from a `Request` by a `Sequence`, passed as arguments to an Operation. Parameters are defined as part of a `Route` using the OpenAPI specification. They can be parsed from the following parts of the `Request`:
+In the example above, `name` is a Parameter. Parameters are values which are
+usually parsed from a `Request` by a `Sequence` and then passed as arguments to
+an Operation. Parameters are defined as part of a `Route` using the OpenAPI
+specification. They can be parsed from the following parts of the `Request`:
 
- - `body`
- - `query` string
- - `header`
- - `path` (url)
+* `body`
+* `form data`
+* `query` string
+* `header`
+* `path` (url)
 
 ## Creating REST Routes
 
 There are three distinct approaches for defining your REST Routes:
-- With an OpenAPI specification object
-- Using partial OpenAPI spec fragments with the `Route` constructor
-- Using route decorators on controller methods
+
+* With an OpenAPI specification object
+* Using partial OpenAPI spec fragments with the `Route` constructor
+* Using route decorators on controller methods
 
 ### Declaring REST Routes with API specifications
 
-Below is an example [Open API Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#swagger-object) that defines the same operation as the example above. This a declarative approach to defining operations. The `x-operation` field in the example below references the handler JavaScript function for the API operation, and should not be confused with `x-operation-name`, which is a string for the Controller method name.
+Below is an example
+[Open API Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#oasObject)
+that defines the same operation as the example above. This a declarative
+approach to defining operations. The `x-operation` field in the example below
```

Comment:
`to defining operations` -> `to define operations`?

Reply:
I might change it to `This is the declarative approach to defining operations`, but what's up there (aside from `is` missing) looks good to me unless I'm horribly off.